pkg/jx/cmd/delete_extension.go

```diff
@@ -172,12 +196,12 @@ func (o *DeleteExtensionOptions) Run() error {
 				return err
 			}
 		}
-		err := extensionsClient.Delete(name, &metav1.DeleteOptions{})
+		err := extensionsClient.Delete(ext.ObjectMeta.Name, &metav1.DeleteOptions{})
 		if err != nil {
```

Comment:
Whilst you're in this area, this will break out of the loop early if an extension fails to delete, which is normally non-ideal.
Eg, if you have `a`, `b`, `c` to delete and `b` fails, `a` will have been deleted, but it won't even attempt to delete `c` once `b` fails.
If this is what you want, then that's OK (but it's normally not)
You probably want to add `err` to a list and return that list-of-errors (or combine them into one error summarising which ones failed to uninstall) and then return that at the end of the `for` loop

Reply:
Makes sense.